tests/Unit/GraphQLQueryTest.php

```diff
@@ -120,6 +120,18 @@ public function testQueryAndReturnResultWithAuthorize(): void
         $this->assertEquals('Unauthorized', $result['errors'][0]['message']);
     }
 
+    /**
+     * Test query with authorize.
+     */
+    public function testQueryAndReturnResultWithAuthorizeMessage(): void
```

Comment:
```suggestion
    public function testQueryAndReturnResultWithCustomAuthorizeMessage(): void
```
What do you think about this suggestion?